src/watcher.js

```diff
@@ -58,7 +59,12 @@ export default class Watcher {
             let tag = this.app.tag || getConfig().tag || 'orca.' + Date.now() //We append a timestamp since we want to make sure Marathon pulls the new image
             let imageName = getConfig().registry + `/${this.app.name}:${tag}`
             this.log(`Building image ${imageName}`)
-            await hostExec('docker', ['build', '-t', imageName, '.'], {cwd: this.app.dir})
+            let params = ['build', '-t', imageName, '.']
+            _.map(this.buildArgs, (value, key) => {
+                params.push('--build-arg')
+                params.push(`${key}="${value}"`)
```

Comment:
Just realized that there's a risk of including bad unescaped arguments here. Can you escape both the key and the value? There must be a module for that you can use.